_posts/2019-04-25_node_version_menager.md

```diff
@@ -0,0 +1,77 @@
+layout:    post
+title:     "Korzyści wynikające użycia Node Version Manager"
+date:      2019-04-25 10:11:00 +0100
+published: false
+author:    kczechowski
+tags:
+    - nvm
+    - node
+
+
+# Krótkie wprowadzenie do nvm
+JavaScript i cały ekosystem z nim związany jest bardzo rozbudowany i wydaje się że wcale nie zamierza przestać, można czasem usłyszeć że tydzień w którym nie powstał nowy framework do JS’a jest tygodniem straconym. Masa bibliotek w różnych wersjach, kolejne języki rozbudowujące możliwości JavaScriptu i zmieniające to jak go postrzegamy np.: TypeScript czy CoffeeScript, do tego jeszcze Node.js i różne silniki w przeglądarkach i mamy całkiem sporą listę i coraz więcej pracy związanej z zarządzaniem tym wszystkim.
+
+Dlatego tym bardziej warto zainteresować się rozwiązaniami które mają na celu ułatwić nam życie i prace z tym związane.
+Jednym z takich rozwiązań jest nvm czyli Node Version Manager, który ma nam pomóc zarządzać wersjami node.js.
+Cel jest prosty umożliwić na szybkie i latwe przeskakiwanie między wersjami node ni wymagając od nas uprawnien administratora.
+
+## Poniżej najważniejsze zalety i wady:
+
+### Zalety:
+\+ instalacja nie wymaga uprawnień root'a
```

Comment:
przecineczki na końcu, a przy ostatnim plusiku kropeczka :)

Reply:
zrobione

Reply:
a może wrzucilibyśmy tu obrazki z plusikiem i minusikiem jak na jirze? To co zrobiłeś kiepsko wygląda, bo każdy element umieściłeś w oddzielnym paragrafie